app/views/questions/new.html.erb

```diff
@@ -5,23 +5,74 @@
         <div class="box">
           <h2 class="title-no-margin center">New Question</h2>
           <br>
-            <%= form_for @question do |f| %>
-              <div class="form-group">
-                <%= f.text_field :text, placeholder: 'Question', class: 'form-control', autofocus: true %>
-              </div>
-              <div class="form-group">
-                <%= f.label :qset_id, 'Qset', {class: 'control-label'} %>
-                <%= f.collection_select(:qset_id, Qset.find(current_user.org).descendants, :id, :name, {}, {class: 'form-control'}) %>
+            <%= form_for(@question) do |f| %>
+              <%= f.text_field :text, placeholder: 'Question',  class: 'form-control', autofocus: true %>
+              <button type="button" class="btn btn-primary btn-theme btn-generate-q" data-toggle="modal" data-target="#myModal">
```

Comment:
(Not necessary, but now we do have a mix of form input coding styles on the page. That's usually a 'code smell'. That said, it's fine with me / clear to most coders, I'd imagine.)